### ISSUE TITLE
[TASK] Bump embedded Rancher to v2.8.x and RKE2 to v1.27.x

### ISSUE THREAD

**Issue by bk201** [harvester/harvester#5041]
**Is your enhancement related to a problem? Please describe.**

We should ship a supported Rancher and RKE2 in v1.3.0.

**Describe the solution you'd like**

- Bump Rancher to v2.8.x.
- Bump RKE2 to v1.27.x. 

Done with 
- [x] https://github.com/harvester/harvester-installer/pull/655

**Describe alternatives you've considered**

**Additional context**

**Comment by bk201** [comment]
Note we also need to bump Rancher/RKE2 in v1.2 and test this upgrade path.
1.2.1 -> v1.2-head -> v1.3-head.

**Comment by w13915984028** [comment]
RKE2 v1.27.10+rke2r1 has been released
https://github.com/rancher/rke2/releases/tag/v1.27.10%2Brke2r1

The v1.27.10 will solve this important issue: https://github.com/harvester/harvester/issues/4633

Harvester has bumped to RKE2 v1.27.10 with https://github.com/harvester/harvester-installer/pull/655

**Comment by TachunLin** [comment]
Verified **fixed** on `v1.3-50a00ad1-head` (24/02/20). Close this issue.

### Result
<details><summary>$\color{green}{\textsf{PASS}}$ Clean install: bump Rancher to 2.8.2 and RKE2 to 1.27.10  $~~$</summary>

```
harv1:~ # kubectl get settings.management.cattle.io server-version
NAME             VALUE
server-version   v2.8.2
harv1:~ # kubectl get nodes
NAME    STATUS   ROLES                       AGE   VERSION
harv1   Ready    control-plane,etcd,master   30m   v1.27.10+rke2r1
```

 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Upgrade path (v1.2 > v1.2-head > v1.3-head): bump Rancher to 2.8.2 and RKE2 to 1.27.10  $~~$</summary>

```
node1:~ # kubectl get settings.management.cattle.io server-version
NAME             VALUE
server-version   v2.8.2
node1:~ # kubectl get nodes
NAME    STATUS   ROLES                       AGE   VERSION
node1   Ready    control-plane,etcd,master   18h   v1.27.10+rke2r1
node2   Ready    control-plane,etcd,master   17h   v1.27.10+rke2r1
node3   Ready    <none>                      17h   v1.27.10+rke2r1

```


 </details>

### Test Information
* Test Environment: Single and three nodes Harvester on baremetal machines
* Harvester version: `v1.3-50a00ad1-head` (24/02/20)

### Verify Steps
<details><summary>Clean and upgrade path </summary>

  1. Access to Harvester master node
  2. Check the Rancher version 
  ```
  kubectl get settings.management.cattle.io server-version
  ```
  3. Check the RKE2 version
  ```
  kubectl get nodes
  ```
</details>
